conference.js

```diff
@@ -21,6 +21,8 @@ const ConferenceErrors = JitsiMeetJS.errors.conference;
 const TrackEvents = JitsiMeetJS.events.track;
 const TrackErrors = JitsiMeetJS.errors.track;
 
+const RecorderErrors = JitsiMeetJS.errors.recorder;
```

Comment:
Are we using this variable somewhere? If not maybe we should not expose this errors from the library.